contrib/scoverage/src/ScoverageModule.scala

```diff
@@ -70,7 +70,7 @@ trait ScoverageModule extends ScalaModule { outer: ScalaModule =>
   def scoverageClasspath = T {
     Lib.resolveDependencies(
       Seq(coursier.LocalRepositories.ivy2Local, MavenRepository("https://repo1.maven.org/maven2")),
-      Lib.depToDependency(_, outer.scalaVersion()),
+      Lib.depToDependency(_, "2.13.1"),
```

Comment:
Wondering why we hardcode this here?

Reply:
because this is the *worker* process, which shares the classpath with the `contrib.scoverage.worker` Mill module. The Mill module has one scalaVersion that doesn't depend on the module it's being used in, similar to how `scalanativelib.worker` is always 2.12 and `scalajslib.worker` is going to be always 2.13 (as of this PR)

Reply:
I guess, I'd like to move all "2.13" ocurences into a val in the Deps object. But I can do that outside of the PR.